cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -293,13 +293,16 @@ fn send_token_from_ethereum_to_penpal() {
 		(PenpalASender::get(), INITIAL_FUND),
 	]);
 
+	let key = CustomizableAssetFromSystemAssetHub::key().to_vec();
+	let value = Location::new(2, [GlobalConsensus(Ethereum { chain_id: CHAIN_ID })]).encode();
+
+	println!("key:{:?}", hex::encode(&key));
+	println!("value:{:?}", hex::encode(&value));
```

Comment:
Do we want to leave these in?

Reply:
Yeah, context in https://github.com/Snowfork/snowbridge/pull/1174/files#r1565989912